src/plugins/GenericPlugin.ts

```diff
@@ -36,6 +36,11 @@ class GenericPlugin implements IPlugin {
 
     this.ad = ad;
   }
+
+  public isEnabled(prop: string): boolean {
+    const value = this.ad.configuration[prop] || true;
```

Comment:
Once we have a case where the default isn't true, we will need to revisit

Reply:
we could pass a default param too like
` this.enEabled('iAmTheProperty', true || false)`